tables/variables.md

```diff
@@ -1 +1,71 @@
+# tables/variables
 
+The `Variables` table displays information concerning the various different `variables` that are stored on the database.
+The following `JOIN` queries can be carried out:
+
+- `geography_type_id` on the [topics](topics.md) table using `topic_id`.
```

Comment:
Does this mean to say `geography_type_id`?

Reply:
nope. fixed